src/main/java/org/interledger/btp/SubProtocolContentType.java

```diff
@@ -0,0 +1,31 @@
+package org.interledger.btp;
+
+public enum SubProtocolContentType {
```

Comment:
I'm still not sure I like the restriction in content-type here...see discussion in [https://github.com/interledger/rfcs/issues/286](https://github.com/interledger/rfcs/issues/286) for more details. 

I suppose for now, this is the spec....